wid/gap.py

```diff
@@ -954,6 +954,18 @@ def hdl_wid_402(desc):
     return True
 
 
+def hdl_wid_402(desc):
```

Comment:
Did you mean 403?

Reply:
Might make sense to call 402 if the expected behavior is the same.

Reply:
yes, it was suppose to be 403 :)

Reply:
one is general, and one is directed,  I'd prefer to keep those separate even if currently implementations match